Clarify beatmap veto conditions (#10476)
ppy/osu-wiki: wiki/People/Beatmap_Nominators/Rules/en.md

Please see the [beatmap veto](/wiki/People/Beatmap_Nominators/Beatmap_Veto) page for full information on the veto process.

- **Do not veto beatmaps of game modes you are not formally qualified for.** You are not allowed to nominate them so blocking a nomination without the ability to renominate the beatmap makes no sense. This does not apply to vetoes regarding audio, backgrounds, or other content that does not directly pertain to the beatmap's game mode.

> - **You can veto a beatmap by posting the issues you think need to be addressed before reaching Ranked status in the form of Problem stamps in the respective modding discussion.** This will reset any nominations the beatmap has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must include a clear problem and reasoning for stopping the ranking process of the beatmap. You must participate in the resulting discussion and attempt to reach a mutual agreement.

Reviewer: ```suggestion
- **You can veto a beatmap by posting the issues you think need to be addressed before reaching Ranked status in the form of problem stamps in the respective modding discussion.** This will reset any nominations the beatmap has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must include a clear problem and reasoning for stopping the ranking process of the beatmap. You must participate in the resulting discussion and attempt to reach a mutual agreement.
```

- **Do not veto for unrankables.** Unrankables imply that no other nominator should nominate and that the mapper has no other choice but to fix it, in the same way an upheld veto works.

> - **Do not nominate vetoed beatmaps.** A beatmap must have its veto dismissed to allow for its renomination, which can happen if the vetoing nominator decides to lift the veto following a satisfactory discussion or appropriate changes to the beatmap, or if the mapper or vetoing nominator requests a mediation and the jury decides that the veto is unnecessary or invalid.

Reviewer: ```suggestion
- **Do not nominate vetoed beatmaps.** A beatmap must have its veto dismissed before renomination, which can happen if the vetoing nominator decides to lift the veto following a satisfactory discussion or appropriate changes to the beatmap, or if the mapper or vetoing nominator requests a mediation and the jury decides that the veto is unnecessary or invalid.
```
